products/zentyal.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Zentyal
+category: os
+tags: linux-distribution
+permalink: /zentyal
+releasePolicyLink: https://zentyal.com/release-policy/
+releaseDateColumn: true
+releaseColumn: false
+eolColumn: Development Edition Support
+eoesColumn: Commercial Support
+
+customColumns:
+-   property: ubuntuVersion
+    position: before-latest-column
```

Comment:
```suggestion
    position: after-release-column
```

Align with what is done on other pages, such as https://endoflife.date/apache-camel.